config/webpack.dev.config.js

```diff
@@ -33,9 +33,11 @@ module.exports = Merge.smart(commonConfig, {
     new webpack.HotModuleReplacementPlugin(),
   ],
   devServer: {
+    host: '0.0.0.0',
+    port: 18011,
     proxy: {
       '/api': {
-        target: 'http://localhost:18010',
+        target: 'http://docker.for.mac.localhost:18010',
```

Comment:
I can't fully test it because my devstack is broken right now (going to destroy and start again), but I'm pretty sure this won't work on my ubuntu machine :penguin: